entity-framework/core/providers/in-memory/index.md

```diff
@@ -12,7 +12,7 @@ uid: core/providers/in-memory/index
 ---
 # EF Core In-Memory Database Provider
 
-This database provider allows Entity Framework Core to be used with an in-memory database. This is useful when testing code that uses Entity Framework Core. The provider is maintained as part of the [EntityFramework GitHub project](https://github.com/aspnet/EntityFramework).
+This database provider allows Entity Framework Core to be used with an in-memory database. This is useful when testing code that uses Entity Framework Core. The provider is maintained as part of the [Entity Framework Core Project](https://github.com/aspnet/EntityFrameworkCore).
```

Comment:
Maybe "This can be useful for testing, although the SQLite provider in in-memory mode may be a more appropriate test replacement for relational databases."